JSON/Simon Forgets.json

```diff
@@ -59,9 +59,6 @@
   "Souvenir": { "Status": "NotACandidate" },
   "SteamID": "1952732783",
   "Symbol": "Sf",
-  "TwitchPlays": {
-    "AutoPin": true,
-    "ScoreExplanation": "This module awards 2 point per correct input plus 3 points for every module on the bomb upon solving."
-  },

```

Comment:
This should not be removed. This is to keep the explanation of how the module is scored as it is not simple.